superset-frontend/src/filters/components/Range/RangeFilterPlugin.tsx

```diff
@@ -118,47 +103,86 @@ export default function RangeFilterPlugin(props: PluginFilterRangeProps) {
   const [row] = data;
   // @ts-ignore
   const { min, max }: { min: number; max: number } = row;
-  const { groupby, defaultValue, inputRef } = formData;
+  const { groupby, defaultValue, inputRef, stepSize, logScale } = formData;
   const [col = ''] = groupby || [];
   const [value, setValue] = useState<[number, number]>(
     defaultValue ?? [min, max],
   );
   const [marks, setMarks] = useState<{ [key: number]: string }>({});
 
+  // these could be replaced with a property instead, to allow custom transforms
+  const transformScale = useCallback(
+    (val: number | null) =>
+      logScale && val ? (val > 0 ? Math.log10(val) : 0) : val,
+    [logScale],
+  );
+
+  const inverseScale = useCallback(
+    (val: number | null) => (logScale && val ? Math.pow(10, val) : val),
+    [logScale],
+  );
```

Comment:
Could be a good idea to break these out into a util and add unit tests for these, just in case (I can see the need later for introducing other than base 10)

Reply:
I think it would be interesting to allow generic transform and inverse functions...somehow... eg, you need quadratic, but I need log. I thought about adding an extra dropdown in the config, to allow the designer to choose between the two, but I left it out because I wasn't sure I'd need it myself.